snuba/clusters/storage_sets.py

```diff
@@ -17,6 +17,7 @@
     "FUNCTIONS": "functions",
     "SEARCH_ISSUES": "search_issues",
     "SPANS": "spans",
+    "SPANS_V2": "spans_v2",
```

Comment:
I think we should make it clear it's a events platform dataset by using `eap_spans` instead of a generic v2.